connectors/github/element-templates/github-webhook-connector-start-event.json

```
@@ -104,15 +104,27 @@
       "description":"Condition under which the connector triggers. <a href='https://docs.camunda.io/docs/components/connectors/out-of-the-box-connectors/github-webhook/' target='_blank'>See documentation</a>"
     },
     {
-      "label":"Variables",
-      "type":"String",
-      "group":"variable-mapping",
-      "feel":"required",
-      "binding":{
-        "type":"zeebe:property",
-        "name":"inbound.variableMapping"
+      "label": "Result Variable",
+      "type": "String",
+      "group": "variable-mapping",
+      "optional": true,
+      "binding": {
+        "type": "zeebe:property",
+        "name": "resultVariable"
+      },
+      "description": "Name of variable to store the result of the Connector in"
+    },
+    {
+      "label":"Result Expression",
```

Let's normilize with other labels

```suggestion
      "label":"Result expression",
```